main/mainTemplate.json

```diff
@@ -6,6 +6,10 @@
       "type": "string",
       "defaultValue": "[deployment().properties.templateLink.uri]"
     },
+    "_artifactsLocationSasToken": {
```

Comment:
add  description

Reply:
SasToken of what?

Reply:
Added description.
We do not use this parameter, but test-tool-kit checks it.